sov-modules/sov-modules-macros/src/dispatch/common.rs

```diff
@@ -173,29 +176,28 @@ pub fn get_serialization_attrs(item: &syn::DeriveInput) -> Result<Vec<TokenStrea
     let mut has_deserialize = false;
     let mut has_other = false;
 
-    let attributes: Vec<String> = 
-        serialization_attrs.iter().map(|t| t.to_string()).collect();        
+    let attributes: Vec<String> = serialization_attrs.iter().map(|t| t.to_string()).collect();
 
-    for attr in &attributes {        
+    for attr in &attributes {
```

Comment:
I guess it is not critical for compile time, but we can check if all 3 booleans are true and then break.

Reply:
Going to save this for future cleanup, since all that changed in this PR was formatting